package.json

```diff
@@ -6,27 +6,18 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "",
-  "license": "ISC",
+  "license": "MIT",
+  "author": "Microsoft Corp.",
   "dependencies": {
-    "az-iot-bi-test": "0.0.2",
-    "az-iot-helper": "^0.1.1",
-    "azure-event-hubs": "0.0.2",
+    "az-iot-bi": "0.1.4",
     "get-gulp-args": "0.0.1",
-    "gulp": "^3.9.1",
     "gulp-help": "^1.6.1",
-    "gulp-log-capture": "0.0.6",
-    "gulp-rsync": "0.0.6",
-    "gulp-run": "^1.7.1",
-    "gulp-shell": "^0.5.2",
     "q": "^1.4.1",
+    "unzip": "^0.1.11",
     "request": "^2.74.0",
     "run-sequence": "^1.2.2",
     "simple-ssh": "^0.9.0",
     "scp2": "^0.5.0",
-    "ssh2": "^0.5.1",
-    "unzip": "^0.1.11",
-    "moment": "^2.15.0",
-    "azure-storage": "^1.3.0"
```

Comment:
These packages are required by azure-table.js .
Please add them back to package.json in az-blink repo.